extensions/db/src/main/java/org/mule/extension/db/api/config/MySqlDbUtils.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.db.api.config;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Map;
+
+/**
+ * Utility class for MySQL Connection
+ *
+ * @since 4.0
+ */
+final class MySqlDbUtils {
```

Comment:
the logic that was moved here applies to other vendor specific connections such as oracle. Why move them to a mysql Specfic utility class?